inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_non_max_suppression.cpp

```diff
@@ -56,7 +56,7 @@ class DynamicToStaticShapeNonMaxSuppression : public CommonTestUtils::TestsCommo
         const auto dims = std::make_shared<ngraph::opset3::Parameter>(ngraph::element::i64, ngraph::Shape{3});
         const auto dsr = std::make_shared<ngraph::vpu::op::DynamicShapeResolver>(scores, dims);
 
-        const auto node = std::make_shared<ngraph::op::v4::NonMaxSuppression>(
+        const auto node = std::make_shared<ngraph::op::dynamic::NonMaxSuppression>(
```

Comment:
We have inference test for NMS here:
inference-engine\tests\functional\plugin\myriad\subgraph_tests\dsr_non_max_suppression.cpp

It is disabled, but still should stay consistent after this PR merged.